extensions/shared/register-jetpack-block.js

```diff
@@ -48,6 +50,14 @@ export default function registerJetpackBlock( name, settings, childBlocks = [] )
 		edit: requiredPlan ? wrapPaidBlock( { requiredPlan } )( settings.edit ) : settings.edit,
 	} );
 
+	if ( requiredPlan ) {
+		addFilter(
+			'editor.BlockListBlock',
+			`jetpack/${ name }-with-has-warning-is-interactive-class-names`,
```

Comment:
Haha that's a mouthful!

Reply:
Yeah, same for the component name :grimacing: Happy to change it something more palatable!